controllers/controller.js

```diff
@@ -41,3 +42,13 @@ exports.getComments = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.postComment = (req, res, next) => {
+  const { article_id } = req.params;
+  const commentObj = req.body;
+  updateComments(article_id, commentObj)
+    .then((newComment) => {
+      res.status(201).send(newComment);
```

Comment:
common practice with JSON to send back the data in a nested object on a descriptive key (will help make dealing with responses in frontend a bit clearer)

 i.e. `{ comment: newComment }`